src/core/audio/audio_manager.hpp

```diff
@@ -0,0 +1,25 @@
+#pragma once
+
+#include <memory>
+
+#include "core/audio/processor.hpp"
+#include "debug/ui.hpp"
+#include "util/event.hpp"
+
+namespace otto::audio {
+  namespace events {
+    util::Event<>& pre_init();
```

Comment:
this is great as a temporary events interface. But its definately something we need to look at.